src/index.html

```diff
@@ -16,17 +19,30 @@
       href="style.css"
     />
   </head>
+
```

Comment:
The `<body>` tag should not be self-closing. It should be written as `<body>` and closed with a separate `</body>` tag at the end of the document.